students/thorn/lesson03/mailroom.py

```diff
@@ -9,9 +9,9 @@
 # Global that holds donor history and the amounts donated.
 DONOR_HISTORY = [
     ['Tom Horn',        [599.23, 1000.00]],
-    ['Theo Hartwell',   [1000.00, 900.88, 23.6]],
-    ['James Jones',     [8723.22, 27167.22, 91817.66]],
-    ['Sterling Shepard',[90012.32, 2312.24]],
+    ['Theo Hartwell',   [0.01, 0.01, 0.1]],
+    ['Bailey Kimmitt',  [8723.22, 27167.22, 91817.66]],
+    ['Paul Hubbell',    [90012.32, 2312.24]],
     ['David Beckham',   [1817266.11, 123123.66, 111335.112]]
```

Comment:
all caps indicates a constant, in our case donor data is not a constant so it should be defined as all lower case.